.github/workflows/prepare-for-minor-release.yml

```diff
@@ -174,3 +185,34 @@ jobs:
             ${{ matrix.previous_version }} only
           add-paths: |
             composer.json
+  update_composer_dev_branch_alias_on_main_branch:
+    needs: [prepare_for_new_minor_release, branch_name_prep]
+    runs-on: ubuntu-latest
+    name: Update the composer dev branch alias on the `main` branch.
+    steps:
+      - name: Update `main` branch with new minor release branch alias.
+        uses: actions/checkout@v4
+        with:
+          ref: main
+          token: ${{ secrets.REPO_DISPATCH_TOKEN }}
+
+      - name: Set up Git
+        run: |
+          git config user.name "GitHub Action"
+          git config user.email "action@github.com"
+
+      - name: Update dev branch alias via Composer
+        run: |
+          composer config extra.branch-alias.dev-main ${{ needs.branch_name_prep.outputs.next_minor_release }}.x-dev
+
+      - name: Create pull request for main branch updates
+        uses: peter-evans/create-pull-request@v6
+        with:
+          title: Prepare `main` branch for new minor release branch ${{ inputs.release_branch_name }}
+          commit-message: Prepare `main` branch for new minor release branch ${{ inputs.release_branch_name }}
+          branch: update-main-for-${{ inputs.release_branch_name }}
+          base: main
```

Comment:
Not sure what label to put here, since we don't have access to the next version number yet.

Reply:
The other step sets a label like this
```
          labels: |
            ${{ matrix.previous_version }} only	            ${{ matrix.previous_version }} only
```

Reply:
Maybe just "task"?